app/styles/_overlay-forms.less

```diff
@@ -1,3 +1,7 @@
+.order-service-config-single-column {
+  width: 100%;
```

Comment:
We previously discussed a plan to refactor the overlay to a simpler layout when we switch to a single column design; therefore, I think this is ready for merge as is.